composer.json

```diff
@@ -13,6 +13,7 @@
     ],
     "require": {
         "php": "^7.1",
+        "symfony/http-kernel": "^3.1 || ^4.0",
```

Comment:
do we really need a dependency on the symfony kernel?
keep in mind that this is a php library rather than a symfony bundle.

Reply:
I second @wickedOne here!